src/screens/AirdropDashboard/components/CommunityGoalsListItem.tsx

```diff
@@ -0,0 +1,85 @@
+import React, { FC } from 'react';
+import { View, StyleSheet, Text } from 'react-native';
+import ProgressBar from 'react-native-progress/Bar';
+
+import { icons } from 'app/assets';
+import { typography, palette } from 'app/styles';
+
+const i18n = require('../../../../loc');
+
+interface CommunityGoalsListItemProps {
+  threshold: number;
+  reward: string;
+  currentUsersQuantity: number;
+  previousThreshold: number;
+}
+
+export const CommunityGoalsListItem: FC<CommunityGoalsListItemProps> = props => {
+  const { threshold, reward, currentUsersQuantity, previousThreshold } = props;
+
+  const localUsersQuantity = currentUsersQuantity - previousThreshold;
+  const localThreshold = threshold - previousThreshold;
+  const locked = localUsersQuantity < 0;
+  const progress = locked ? 0 : localUsersQuantity / localThreshold;
+
+  return (
+    <View style={styles.container}>
+      <View style={styles.headerContainer}>
+        {locked && <icons.LockIconGrey width={10} height={13} style={styles.lockImage} />}
+        <Text style={[styles.header, ...[locked ? [styles.lockedText] : []]]}>
```

Comment:
could be written as `[styles.header, locked ? styles.lockedText : null]`

Reply:
This way there s no spare elements at all. In your version null will still be assigned to array. It might not matter in the end, but it s cleaner in a way.

Reply:
keeping as is. done